### PR TITLE
Add option to return json on websocket

### PR DESCRIPTION
**Pull request by cceyda**
I have added the option to also get the results as a json formatted string from the websocket by setting `ws.send_str('json')`.
json is more convenient for consuming in other applications, that are piggy-backing off of this library.

I have been using your gpustat & gpustat-web libraries for a while 😄 
~ Thank you ~

### CODE REVIEW

File: gpustat_web/app.py [hunk 1]
```diff
@@ -161,6 +162,18 @@ def render_gpustat_body():
...
 
+def render_gpustat_body_json():
+    body = []
+    for host, status in context.host_status.items():
+        if not status:
+            continue
+        try:
+            parsed = json.loads(status)
+        except:
+            continue
```
Comment (wookayin) [inline gpustat_web/app.py:177]: Please handle exception properly.

**Comment by wookayin** [comment]
May I ask what is the main purpose of emitting the data json? All the data is given as 'text' actually too.

**Comment by cceyda** [comment]
the ansi conversion was making the text hard to parse in the websocket response

**Comment by wookayin** [comment]
I still don't quite understand. The JSON response is an array of still ANSI-coded gpustat output, right? Why do you need them in the first place?

**Comment by cceyda** [comment]
Because otherwise websocket returns info formatted with html (with tags, and some stats left out),
due to conversion done here:
`return ansi_conv.convert(body, full=False)`
the response looks like this:
```html
<span class="ansi36">[0]</span> <span class="ansi34">TITAN RTX                </span> |<span class="ansi31"> 26°C</span>, <span class="ansi32">  0 %</span> | <span class="ansi36"></span><span class="ansi1 ansi36"></span><span class="ansi1 ansi33">    1</span> / <span class="ansi33">24220</span> MB | ...
```

I want the whole gpu info in json format, like this:
```json
[{"hostname": "server1", "query_time": "2021-06-26T17:39:18.868461", "gpus": [{"index": 0, "uuid": ....
```
So that I can do my own parsing and formatting.

**Review by chill-otter — APPROVED** [review]

**Review by wookayin — CHANGES_REQUESTED** [review]
I am not sure if the JSON format here is the best possible one. I think instead of having a list as a top-level element, we should have some dict(object) a la gpustat's JSON output.
